### DISCUSSION TITLE
Is mackie control supported in mido?

### DISCUSSION THREAD

**Discussion by titojff** [mido/mido#473] [Q&A]
Is mackie control supported in mido?

**Comment by rdoursenaud** [comment]
Hi!
The Mackie Control protocol uses standard MIDI and some System Exclusive messages, both of which are supported by `mido`.
I have helped documenting the protocol at https://github.com/EMATech/TouchMCU/blob/main/doc/mackie_control_protocol.md
If you need a higher level of abstraction, you will have to roll your own library.
